Feature/installdrupal

lorrx opened the pull request:
Installed Acquias Lightning project.
Installed required composer packages

PierreRe commented:
Have we specified an official board  language yet? English or German?

Ich würde den geheimen DB Hash String zur Sicherheit noch einmal ändern und dann in eine Servervariable auslagern und aus der settings.php entfernen, so dass keine Referenz im öffentlichen Repo auftaucht.
Deine Meinung?

lorrx commented:
I would prefer English to learn the language. Please correct my mistakes ;-)
I've changed and moved the hash_salt property into the secure file.

Greez
